### ISSUE TITLE
binanceus fetch_order() returns no fee/fees structure

### ISSUE THREAD

**Issue by RoscoeTheDog** [ccxt/ccxt#16865]
### ؜

After placing a market order on any trading pair that is NOT BTC or ETH (I know these are 0% trading fees rn), an order response is returned with the correct fee information like so:
```
{'currency': 'BNB', 'cost': 0.00015476}
```
However after passing the order ID to `fetch_orders(id)`, I get the verified order, but without any fee information:
```
None
[]
```

This was tested on the SLP/USDT market with a market order. Although I'm sure it is present in other markets as well I just haven't tested it yet.

Is this one of those cases where I have to manually handle it, or is there a broader solution to this? It seems weird that the original order would contain fee information but fetching it from the exchange would return None.

**Comment by RoscoeTheDog** [comment]
PS This is Windows 10 and latest update of ccxt. I had an issue just another day or two ago so I made sure to be updated to latest version of ccxt. thnx!

**Comment by sc0Vu** [comment]
Hi @RoscoeTheDog

Would you like to paste request / response (remember to remove sensitive information)?

**Comment by RoscoeTheDog** [comment]
Not sure exactly what is sensitive here, but I removed clientorderID which I think might have been a sensitive hash. Either way, this is just a tester API key without withdrawal and only a tiny bit of play money while I build my bots. If there's anything else I should remove let me know and I'll edit this comment asap.

`place_order` response:

```
{'info': {'symbol': 'SLPUSDT', 'orderId': '3085768', 'orderListId': '-1', 'transactTime': '1676536441724', 'price': '0.00000000', 'origQty': '3571.00000000', 'executedQty': '3571.00000000', 'cummulativeQuoteQty': '11.07010000', 'status': 'FILLED', 'timeInForce': 'GTC', 'type': 'MARKET', 'side': 'BUY', 'workingTime': '1676536441724', 'fills': [{'price': '0.00310000', 'qty': '3571.00000000', 'commission': '21.42600000', 'commissionAsset': 'SLP', 'tradeId': '140728'}], 'selfTradePreventionMode': 'EXPIRE_MAKER'}, 'id': '3085768', 'timestamp': 1676536441724, 'datetime': '2023-02-16T08:34:01.724Z', 'lastTradeTimestamp': 1676536441724, 'symbol': 'SLP/USDT', 'type': 'market', 'timeInForce': 'GTC', 'postOnly': False, 'reduceOnly': None, 'side': 'buy', 'price': 0.0031, 'triggerPrice': None, 'amount': 3571.0, 'cost': 11.0701, 'average': 0.0031, 'filled': 3571.0, 'remaining': 0.0, 'status': 'closed', 'fee': {'currency': 'SLP', 'cost': 21.426}, 'trades': [{'info': {'price': '0.00310000', 'qty': '3571.00000000', 'commission': '21.42600000', 'commissionAsset': 'SLP', 'tradeId': '140728'}, 'timestamp': None, 'datetime': None, 'symbol': 'SLP/USDT', 'id': '140728', 'order': None, 'type': None, 'side': None, 'takerOrMaker': None, 'price': 0.0031, 'amount': 3571.0, 'cost': 11.0701, 'fee': {'cost': 21.426, 'currency': 'SLP'}, 'fees': [{'cost': 21.426, 'currency': 'SLP'}]}], 'fees': [{'currency': 'SLP', 'cost': 21.426}], 'stopPrice': None}
```

You can see how 'fees' exist in the place order above.

But when taking said order ID and passing it to fetch_order(), I get this response:
```
{'info': {'symbol': 'SLPUSDT', 'orderId': '3085768', 'orderListId': '-1', 'price': '0.00000000', 'origQty': '3571.00000000', 'executedQty': '3571.00000000', 'cummulativeQuoteQty': '11.07010000', 'status': 'FILLED', 'timeInForce': 'GTC', 'type': 'MARKET', 'side': 'BUY', 'stopPrice': '0.00000000', 'icebergQty': '0.00000000', 'time': '1676536441724', 'updateTime': '1676536441724', 'isWorking': True, 'workingTime': '1676536441724', 'origQuoteOrderQty': '0.00000000', 'selfTradePreventionMode': 'EXPIRE_MAKER'}, 'id': '3085768', 'timestamp': 1676536441724, 'datetime': '2023-02-16T08:34:01.724Z', 'lastTradeTimestamp': None, 'symbol': 'SLP/USDT', 'type': 'market', 'timeInForce': 'GTC', 'postOnly': False, 'reduceOnly': None, 'side': 'buy', 'price': 0.0031, 'triggerPrice': None, 'amount': 3571.0, 'cost': 11.0701, 'average': 0.0031, 'filled': 3571.0, 'remaining': 0.0, 'status': 'closed', 'fee': None, 'trades': [], 'fees': [], 'stopPrice': None}
```

**Comment by kroitor** [comment]
@RoscoeTheDog in general, fees are properties of trades, not orders: https://docs.ccxt.com/en/latest/manual.html#how-orders-are-related-to-trades 

If the exchange API does not provide the fee info from one of the endpoints, CCXT will not invent that data out of thin air. You can verify that by looking into the `info` section of the response you posted above – that is the info from the exchange as is, first hand, untouched. Most of the exchanges will not show the actual settled fees on orders, but will show the fees with `fetch_my_trades` (you can filter them by an order id after fetching them).

Let us know if that does not answer your question. Feel free to reopen this issue if that does not help or just ask further questions if any.
